decidim-core/config/locales/fr.yml

```
@@ -376,7 +376,7 @@ fr:
         name: Bannière en dessous de l'image principale ("hero")
     core:
       actions:
-        login_before_access: Veuillez vous connecter avec votre compte avant de réaliser cette action
+        login_before_access: decidim-initiatives/app/models/decidim/initiative.rb Veuillez vous connecter avec votre compte avant de réaliser cette action
```

```suggestion
        login_before_access: Veuillez vous connecter avec votre compte avant de réaliser cette action
```
Hi, is this translation voluntary ?

---

No, I will double check all.